pkg/operator/starter.go

```diff
@@ -0,0 +1,34 @@
+package operator
+
+import (
+	"context"
+	"time"
+
+	configv1client "github.com/openshift/client-go/config/clientset/versioned"
+	"github.com/openshift/library-go/pkg/controller/controllercmd"
+
```

Comment:
spaces without value!  I threaten you with ascii art!

Reply:
oh :)